test/jdk/java/math/BigInteger/TestDivWord.java

```diff
@@ -0,0 +1,56 @@
+import java.math.Accessor;
+import java.time.Duration;
+import java.util.Random;
+
+public class TestDivWord {
```

Comment:
Where is this used in actually jtreg testing?

Reply:
@bplb It is not used in jtreg testing.

Reply:
So it is only for verification purposes in the context of this PR?

Reply:
@bplb Yes, it is.